### ISSUE TITLE
Thought you might be interested

### ISSUE THREAD

**Issue by MadMax389** [gaugo87/PIXEstL#4]
I was able to create my own 7 layer color profile of the 4 Bambu CMYK filaments. Printing 7 layers at .07mm instead of 5 layers at .1mm gives a good color blend and resolution. The quality is almost the same as a .2mm nozzle by using a .4mm nozzle with .3mm line width setting. Thanks for this awesome program!

![Post 2](https://github.com/gaugo87/PIXEstL/assets/96731892/5d7d9177-77a7-4101-a8c0-e9133223d40b)
![Post 3](https://github.com/gaugo87/PIXEstL/assets/96731892/b5bbe42b-c1fb-4576-b109-3f43bc6a501e)
![Post 4](https://github.com/gaugo87/PIXEstL/assets/96731892/5107906a-2ac7-49b2-99a0-f7e0036fc17d)
![Post 1](https://github.com/gaugo87/PIXEstL/assets/96731892/0682e4bf-1d6b-4792-9d20-4d2b1a011aa8)

**Comment by gaugo87** [comment]
Wow this is very impressive!

I hadn't thought about making such thin layers to achieve these pale colors. I didn't think the printer was capable of that.

It is indeed a good alternative for those who have only one AMS. I'm going to study that. 
On the other hand, with regard to polygons, it must make huge files...

Thanks Madmax.

**Comment by MadMax389** [comment]
Yes, they are fairly large. For the 115mm hex, the total of the stls is about 1GB. It takes a minute to load and slice, but they print in about 5 hours. X1C and the A1 Mini have no problems with .07mm. I tried 9 layers at .05mm, but it was diminishing returns regarding size and print time.

**Comment by monkeyfingerslocksport** [comment]
Wow, I will now be redoing my setting to try and emulate this.

**Comment by arunas-smala** [comment]
@MadMax389 how did you make hexagon? By croping each STL afterwards?

**Comment by MadMax389** [comment]
> @MadMax389 how did you make hexagon? By croping each STL afterwards?

No, I made a "mask" shape in Fusion and overlay the whole lithophane with it as a negative volume. You could do any shape this way.
![image](https://github.com/gaugo87/PIXEstL/assets/96731892/72da6ca7-85fa-4852-a5b7-4a7dcd28e47f)
![image](https://github.com/gaugo87/PIXEstL/assets/96731892/3fbe7f6e-4bc6-489a-96b5-6020bc6052c6)

**Comment by MadMax389** [comment]
> Wow, I will now be redoing my setting to try and emulate this.
> 
> would you mind sharing your filament-palette file.
My results may not be typical as I've tuned these for my display and light source. I created a 7 layer swatch at .07mm layer height, color corrected the image and then analyzed each color at each layer for the hsl values. I've attached my 7 layer swatch. Load them as an assembly, then assign your colors to each stl.
![image](https://github.com/gaugo87/PIXEstL/assets/96731892/30986da1-0a3a-4da7-acd1-32e340ee02e1)
[7 Layer Swatch.zip](https://github.com/gaugo87/PIXEstL/files/13771571/7.Layer.Swatch.zip)

**Comment by monkeyfingerslocksport** [comment]
![test swatch](https://github.com/gaugo87/PIXEstL/assets/154732953/bea731ae-8a1f-474b-baea-d1ab9fdcb936)
Thank you for sharing this is the stl I made it's similar to yours .  My problem is i think i made an error in my filament-palette file as when I try to create a new stl is just crashes out.  
 
[colour swatch pixestl.zip](https://github.com/gaugo87/PIXEstL/files/13774031/colour.swatch.pixestl.zip)

**Comment by MadMax389** [comment]
Yeah, I had to be real careful adding filaments to the .json. One missed bracket or comma can cause problems. I made an excel spreadsheet as an interface where I could select parameters and it will generate the command line.

**Comment by monkeyfingerslocksport** [comment]
![1 x 3 bird](https://github.com/gaugo87/PIXEstL/assets/154732953/ff6f1d9a-11bc-49ca-b492-007ad884b1ab)
![0 7x 3 bird](https://github.com/gaugo87/PIXEstL/assets/154732953/86b3d4e2-e403-4d83-9429-efba6563940e)
both 0.3 line but 1 is 0.10 layer the other is 0.07 it like night and day .. 5 hour for the first and 7.5 hours to print the second

**Comment by MadMax389** [comment]
That looks great! What filaments?

**Comment by monkeyfingerslocksport** [comment]
Both Bambu cymk set

**Comment by qszhuan** [comment]
@monkeyfingerslocksport do you mind to share your palette? thanks! I am also using Bambu CMYK set.

**Comment by monkeyfingerslocksport** [comment]
[Uploading filament-palette-0.07mm.json…]()

**Comment by qszhuan** [comment]
> Uploading filament-palette-0.07mm.json…

Never mind. I printed the swatch, will find out.

**Comment by dts350z** [comment]
> Uploading filament-palette-0.07mm.json…

That link just points back to this issue. Is your .json posted somewhere?

**Comment by guyversama** [comment]
hello, this is absolutely incredible! 
I'd love to try those  settings, would you be ok with sharing your files ? 
I'm nowhere near good enough to understand how to make it myself.

**Comment by derchrome** [comment]
Could you please share your 0.7 JSON file?
